docs/testing.md

```diff
@@ -0,0 +1,94 @@
+# Running the Automated Tests
+
+## JavaScript
+
+### Validating JavaScript
```

Comment:
Do you we have info about the use of `yarn format` for prettier formatting? I forget about it sometimes until Travis yells at me.

Reply:
Good call.  Doesn't hurt to add it here, too.